core/runtime/wasm_edge/module_factory_impl.hpp

```diff
@@ -43,7 +43,9 @@ namespace kagome::runtime::wasm_edge {
     };
     struct Config {
       Config(ExecType exec)
-          : exec{exec}, compiled_module_dir{"/tmp/kagome/wasm-edge"} {}
+          : exec{exec},
+            // TODO(Harrm): use app config
```

Comment:
It is expected that this will either have an issue id or be fixed before the pr is merged.